@viewsort/viewsort.m

```diff
@@ -78,19 +78,19 @@
     data.Noise = 1/(hdf5read(Args.HMMFile,Args.HMMNoise));
     cd(cwd);
     
+    % get and plot ISI coefficient of variation for waveform
+    data.coeffV_ISI = zeros(sf1,1);
     if (isfield(l,'mlseq') > 0) % skip incomplete files
-        % get and plot ISI coefficient of variation for waveform
         spikeSort = l.mlseq;
         for ind = 1:sf1
             [M,I] = min(data.spikeForms(ind,:)); % get index of peak
             spikeTimes = find(spikeSort(ind,:) == I); % get index of peaks
             spikeTimes = spikeTimes/30; spike_ISI = diff(spikeTimes);
-            data.coeffV_ISI(ind) = std(spike_ISI)/mean(spike_ISI);
+            data.coeffV_ISI(ind,1) = std(spike_ISI)/mean(spike_ISI);
```

Comment:
Why is this necessary? data.coeffV_ISI should be a column vector, so there should not be a need to add a column index.

Reply:
This is the latest comment.

Reply:
This is the latest comment.